cetmix_tower_aws/readme/USAGE.md

```
@@ -1,24 +1,33 @@
+- Select `Action`: "Execute Python code"
+- Set `Access Level`: Choose appropriate level (e.g., "Manager")
+- Optional: Set `Default Path` if needed
+- Optional: Add `Tags` (e.g., "aws", "ec2") for better organization
 
-* **Configure Command Settings**
-   - Set a descriptive `Name` (e.g., "List AWS EC2 Instances")
-   - Leave `Reference` blank to generate automatically (or set a custom reference)
-   - Select `Action`: "Run Python code"
-   - Set `Access Level`: Choose appropriate level (e.g., "Manager")
-   - Optional: Add `Tags` (e.g., "aws", "ec2") for better organization
+### Add Required Variables
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)